xarray/core/utils.py

```diff
@@ -9,6 +9,7 @@
 import re
 import warnings
 from collections import Mapping, MutableMapping, Iterable
+from six.moves import cPickle as pickle
```

Comment:
Wow sorry, I actually forgot it wasn't a builtin, and I lazily didn't set up a proper dev environment.  Definitely a simple fix.